packages/error-reporting/src/interfaces/manual.js

```diff
@@ -74,6 +74,10 @@ function handlerSetup(client, config) {
     }
 
     if (err instanceof ErrorMessage) {
+      var fauxError = new Error(err.message);
+      var fullStack = fauxError.stack.split('\n');
+      var cleanedStack = fullStack.slice(0, 1).concat(fullStack.slice(2)).join('\n');
```

Comment:
This matches what is done elsewhere in the code, but I'll add a comment for clarity.